eq-author-api/schema/resolvers/logic/routing2/index.js

```diff
@@ -1,17 +1,11 @@
 const Resolvers = {};
 
-const { find, flatMap } = require("lodash/fp");
+const { getPageById, getSectionById } = require("../../utils");
 
 Resolvers.Destination2 = {
-  page: ({ pageId }, args, ctx) =>
-    pageId
-      ? find(
-          { id: pageId },
-          flatMap(section => section.pages, ctx.questionnaire.sections)
-        )
-      : null,
+  page: ({ pageId }, args, ctx) => (pageId ? getPageById(ctx, pageId) : null),
   section: ({ sectionId }, args, ctx) =>
-    sectionId ? find({ id: sectionId }, ctx.questionnaire.sections) : null,
+    sectionId ? getSectionById(ctx, sectionId) : null,
 };
```

Comment:
How would you feel about this throwing an error instead of returning null? Realistically, the error state should never be reached as the section will (or should) always exist.  For that same reason, if the section can't be found by it's ID, then something is wrong and will need looking at. Returning null could accidentally hide some errors.

Reply:
I'll double check in the morning but it could be useful. There were sometimes when I was working on it that I was accidentally adding the wrong stuff to the wrong place. Only after looking at the network requests could I see that I was adding the wrong info.

Reply:
I think trying to add anything other than `null` actually causes it to error in itself. When changing it from `null` you get a pretty decent stacktrace coming from it